app/src/widgets/MessageList/ui/MessageList.tsx

```diff
@@ -96,11 +97,13 @@ const MessageList: FC = () => {
         setValueInput(event.target.value);
     }, []);
 
+    const { t } = useTranslation();
+
     return (
         <>
             <Input
                 variant={InputVariants.FILLED}
-                placeholder="Поиск..." /*TODO: Перевод - Translate*/
+                placeholder={t('home.search.title')}
```

Comment:
не не не 
это компонент MessageList он может оказаться не только в Home
в словаре нужно сделать отдельной 

```suggestion
                placeholder={t('message-list.search.title')}
```